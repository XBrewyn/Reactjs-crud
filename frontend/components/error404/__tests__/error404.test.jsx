import React from 'react';
import { create } from 'react-test-renderer';
import Error404 from '..';

describe('<Error404 />', () => {
  it('Renders correctly', () => {
    const component = create(<Error404 />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
