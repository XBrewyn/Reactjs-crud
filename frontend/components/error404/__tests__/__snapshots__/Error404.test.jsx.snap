// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error404 /> Renders correctly 1`] = `
<div
  className="css-145grdo"
>
  <div
    className="sketch"
  >
    <div
      className="bee-sketch red"
    />
    <div
      className="bee-sketch blue"
    />
  </div>
  <h1>
    404:
    <small>
      Players Not Found
    </small>
  </h1>
</div>
`;
